scripts/qa_crowbarsetup.sh

```diff
@@ -2520,6 +2520,20 @@ function oncontroller()
     return $?
 }
 
+function install_suse_ca()
+{
+    # trust build key - workaround https://bugzilla.opensuse.org/show_bug.cgi?id=935020
+    wget --no-check-certificate -O build.suse.de.key.pgp https://api.suse.de/public/source/SUSE/_pubkey
+    safely sha1sum -c <<EOF
+ee896d59206e451d563fcecef72608546bf10ad6  build.suse.de.key.pgp
+EOF
+    rpm --import build.suse.de.key.pgp
+
```

Comment:
We need to be sure that we are adding the correct repo, so using --auto-import-keys is not an option.